setup.py

```diff
@@ -81,7 +81,7 @@ def get_git_sha() -> str:
         "colorama",
         "croniter>=0.3.28",
         "cron-descriptor",
-        "cryptography>=41.0.0, <41.0.2",
+        "cryptography>=41.0.2",
```

Comment:
Can we keep the upper bound to only allow patch versions?

```suggestion
        "cryptography>=41.0.2, <41.1.0",
```

Reply:
patch versions is pretty aggressive, cryptography seems to follow semantic versioning so minors should be safe also.

Reply:
We had problems in past where minor upgrades broke things (`prophet` is one example) but if `cryptography` is safer then I'm fine changing the upper bound 👍🏼